hazelcast-jet-core/src/main/java/com/hazelcast/jet/windowing/WindowOperations.java

```diff
@@ -62,7 +62,7 @@ private WindowOperations() {
     /**
      * Returns an operation that counts the items in the window.
      */
-    public static <T> WindowOperation<T, ?, Long> summingToLong(@Nonnull Distributed.ToLongFunction<T> mapper) {
+    public static <T> WindowOperation<T, ?, Long> summingToLong(@Nonnull Distributed.ToLongFunction<? super T> mapper) {
```

Comment:
Was there an actual problem with just `T` here? I don't see how `T` could be inferred as anything else but the `mapper`'s type.